README.adoc

```diff
@@ -36,3 +36,6 @@ These are optional settings that can be manually added to the page URL to contro
  ** Disable comment count loading because it's a bit expensive. e.g. `comments=false`
  * `autocomments`
  ** Enable auto comments when making changes to bugs. e.g. `autocomment=true`
+ * `picture`
+ ** Use a picture site option.
+ 1. gravatar
```

Comment:
Hey, if you're calling it, I'm all for less is more. Gravatar is pretty easy to implement, and I'm very doubtful of using social media api's. 
Google OATH is a tricky beast, each person requesting their own API key and such. Third party solutions are available, http://www.avatarapi.com/

I like how you mentioned adding private work emails into Gravatar works nicely for our use case, and very easy for the individual to set it up.